packages/config/index.js

```diff
@@ -408,10 +408,18 @@ function getLevelByConfigs(pathToLevel, options, allConfigs, root) {
  * @returns {Array<{layer: String, path: String}>}
  */
 function extendConfigsPathByLayer(configs) {
+    let levels;
+
     configs.forEach(config => {
-        config.levels && config.levels.forEach(level => {
-            level.path || (level.path = level.layer + '.blocks');
-        });
+        levels = config.levels;
+
+        if (!levels) return;
+
+        if (Array.isArray(levels)) {
+            levels.forEach(level => level.path || (level.path = level.layer + '.blocks'));
```

Comment:
Why we need to add `.blocks`?

Reply:
I just added support for the old levels version.
Frankly speaking, I don't want to change the signature of the intended new version levels here

Reply:
if (!Array.isArray(levels)) {